source/scripts/model/Monster.js

```diff
@@ -100,6 +100,49 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit(magnitude) {
+
+          //Variables for the monster and hero positions
+       var monX = this.position.x
+       var monY = this.position.y
+
+       var adX = this.game.adventurer.position.x 
+       var adY = this.game.adventurer.position.y
```

Comment:
I'm fine with dx and dy being variables, but I've never been a fan of "alias-ing" variable names just because they're longer. adX and monY require more thinking to remember what they are then the more verbose adventurer.position.x and monster.position.y